modules/rediadsBidAdapter.js

```diff
@@ -0,0 +1,103 @@
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { deepSetValue, logWarn, logError } from '../src/utils.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+
+const BIDDER_CODE = 'rediads';
+const ENDPOINT_URL = 'https://bidding.rediads.com/openrtb2/auction';
+const STAGING_ENDPOINT_URL = 'https://stagingbidding.rediads.com/openrtb2/auction';
+const DEFAULT_CURRENCY = 'USD';
+const LOG_PREFIX = 'Rediads: ';
+
+const MEDIA_TYPES = {
+  [BANNER]: 1,
+  [VIDEO]: 2,
+  [NATIVE]: 4,
+};
+
+const converter = ortbConverter({
+  context: {
+    netRevenue: true,
+    ttl: 300,
+    currency: DEFAULT_CURRENCY,
+  },
+  bidResponse(buildBidResponse, bid, context) {
+    let mediaType = 'banner'; // Default media type
+
+    if (bid.vastXml || bid.vastUrl || (bid.adm && bid.adm.startsWith('<VAST'))) {
+      mediaType = 'video';
+    } else if (bid.adm && bid.adm.includes('"native"') && bid.adm.includes('"assets"')) {
+      mediaType = 'native';
+    }
+
+    bid.mediaType = mediaType;
+    bid.mtype = MEDIA_TYPES[mediaType];
+
+    if (bid.mediaType === BANNER) {
+      bid.ad = bid.adm;
+    }
+    return buildBidResponse(bid, context);
+  },
+});
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [NATIVE, BANNER, VIDEO],
+  isBidRequestValid: function (bid) {
+    let isValid = false;
+    const accountID = bid?.params?.account_id
+    if (accountID && typeof accountID === 'string') {
+      isValid = true;
+    } else {
+      logError(`${LOG_PREFIX} account_id is missing from params or is not of type "string"`)
+    }
+    return isValid;
+  },
+  buildRequests(bidRequests, bidderRequest) {
+    const params = bidRequests[0]?.params || {};
+    let data = {};
+    let FINAL_ENDPOINT_URL = params.endpoint || ENDPOINT_URL
+    try {
+      data = converter.toORTB({ bidRequests, bidderRequest });
+      const testBidsRequested = location.hash.includes('rediads-test-bids');
+      const stagingEnvRequested = location.hash.includes('rediads-staging');
+
+      if (stagingEnvRequested) {
+        FINAL_ENDPOINT_URL = STAGING_ENDPOINT_URL;
+      }
+
+      deepSetValue(data, 'ext.rediads.params', params);
+      deepSetValue(data, 'site.content', config.getConfig('content'));
```

Comment:
shouldn't need to import config here. This should be coming from the bid request.

Reply:
@ChrisHuie , We've addressed the review point by removing the config import. The `site.content` value is now sourced directly from the bid request, ensuring it aligns with the expected design. Let us know if you need any further changes!